src/Model/GroundWaterEnergy/gwe-uze.f90

```diff
@@ -1293,6 +1311,61 @@ subroutine uze_bd_obs(this, obstypeid, jj, v, found)
     end select
   end subroutine uze_bd_obs
 
+  !> @brief Stop simulation when UZF object area is not equal to the cell area
+  !!
+  !! GWE equilibrates the temperature of a UZE object with the host cell. UZE
+  !! assumes that the area associated with a specific UZE object is equal to
+  !! the area of the host cell. When this condition is not true, the code
+  !! exits with an appropriate message.
+  !<
+  subroutine uzarea_chk(this, nuz, idxbudgwf)
+    ! -- modules
+    use ConstantsModule, only: IZERO
+    use MathUtilModule, only: is_close
+    ! -- dummy
+    class(GweUzeType) :: this
+    integer(I4B) :: nuz
+    integer(I4B) :: idxbudgwf
+    ! -- local
+    integer(I4B) :: n
+    integer(I4B) :: igwfnode
+    real(DP) :: carea
+    real(DP) :: uzarea
+
+    ! cycle through uze objects, stop at first occurrence of more than one
+    ! uze object in a cell
+    do n = 1, nuz
+      igwfnode = this%flowbudptr%budterm(idxbudgwf)%id2(n)
+      carea = this%dis%area(igwfnode)
+      uzarea = this%flowbudptr%budterm(idxbudgwf)%auxvar(1, n)
+      ! compare areas
+      if (.not. is_close(carea, uzarea)) then
+        call this%print_uz_err(igwfnode)
```

Comment:
Instead of calling `store_error` and `StoreErrorUnit` from this routine, let's keep submitting all the errors, and then after this loop, call `count_errors` and then call `store_error` and `StoreErrorUnit` if any errors were encountered.  Maybe also rename `print_uz_err` to `area_error(igwnode)`.

Reply:
thanks for catching this, addressed in 189d4e6